packages/app-store/jitsivideo/api/add.ts

```diff
@@ -39,5 +40,5 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     }
     return res.status(500);
   }
-  return res.status(200).json({ url: "/apps/installed" });
+  return res.status(200).json({ url: getInstalledAppPath({ variant: "conferencing", slug: "jitsi" }) });
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.